packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -111,7 +114,11 @@ class DriveCommand extends RunCommandBase {
           'Works only if \'browser-name\' is set to \'android-chrome\'')
       ..addOption('chrome-binary',
         help: 'Location of Chrome binary. '
-          'Works only if \'browser-name\' is set to \'chrome\'');
+          'Works only if \'browser-name\' is set to \'chrome\'')
```

Comment:
@liyuqian would it be easier if the command required you to pass in a destination file?

like --write-sksl-on-exit=foo.sksl.json ?

Reply:
That sounds good to me. It also makes my test easier as I don't have to worry about whether the tool will create sksl_02.json or sksl_01.json. With `--write-sksl-on-exit=foo.sksl.json `, I'd expect the file to be overwritten, right?

Reply:
Yup!